### DISCUSSION TITLE
I Signed-Up for Codespaces Beta, Now What?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#3838] [Codespaces]
I've been waiting a Couple Days for Codespaces Beta, and I'm wondering why I don't have it yet, I'm Patient I just need to know the Approximate Time for it to be Released for me

**Comment by 2percentsilk** [comment]
Thanks for your feedback. We know there's a lot of excitement for the beta (and we're very excited to share it with you!) but we're being cautious with our rollout right now so I ask that you hang tight. We cannot provide an ETA for enablement.
